libs/libc/netdb/lib_dnsquery.c

```diff
@@ -110,6 +110,133 @@ struct dns_query_data_s
  * Private Functions
  ****************************************************************************/
 
+/****************************************************************************
+ * Name: stream_send
```

Comment:
Please add Description as in the existing functions here

Reply:
done